frontend/packages/component-library/src/dropdown/customDropdown.module.scss

```diff
@@ -268,6 +268,11 @@
     color: var(--neutral-base-black);
     border-color: var(--neutral-base-black);
     background-color: var(--neutral-base-white);
+
+    &.activeDropdownButton {
```

Comment:
nit: you can also simplify it by changing it to this pattern:
```
&:has(.dropdownButton:hover), &:has(.activeDropdownButton), {
      dropdownButton:not(:disabled) {
      color: var(--neutral-base-black);
      background-color: var(--neutral-gray-10);
    }
}
```

Since &:has(.dropdownButton:hover) actually represents dropdownTriggerButton hover styles and in case we'll need to update hover styles we'll only need to update it in one instance instead of two